cli/cmd/lql.go

```diff
@@ -428,8 +428,13 @@ func runQueryByID(id string, args []api.ExecuteQueryArgument) (
 	error,
 ) {
 	cli.Log.Debugw("running query", "query", id)
-
-	cli.StartProgress(" Executing query...")
+	msg := "Executing query"
+	startTime, startErr := time.Parse(time.RFC3339, args[0].Value)
+	endTime, endErr := time.Parse(time.RFC3339, args[1].Value)
```

Comment:
I feel the way this function is written allows us to make easy mistakes such as 1) having the arguments in a different order and getting the start/end time wrong, and 2) passing less than two arguments which will throw a panic trying to access elements that don't exist since we are not checking that the array has at least two elements.

There are no unit tests for this function. 

@hazedav @dmurray-lacework let's improve this code so we don't leave room for human mistakes. It can be done with further PRs, but let us not forget about this. Please.